src/contractTest/resources/application.yaml

```diff
@@ -104,8 +104,11 @@ uk:
             CA_APPLICANT_SOLICITOR_RES_NOTIFICATION: a08975e3-6438-4f58-a0f6-827c0dcab291
             CA_RESPONDENT_SOLICITOR_RES_NOTIFICATION: b6224e6a-f55c-46f8-bfc7-631d273070a4
             CA_CITIZEN_RES_NOTIFICATION: a34023e7-c545-4352-b509-574d15c84f52
+            C100_AWAITING_PAYMENT: 3cf67615-8b67-4814-9e61-1ce786b986a1
```

Comment:
Change this to CA_AWAITING_PAYMENT as the scenario for both

Reply:
Done Alok.